knative-operator/pkg/apis/operator/v1alpha1/knativekafka_types.go

```diff
@@ -69,3 +69,19 @@ type Channel struct {
 func init() {
 	SchemeBuilder.Register(&KnativeKafka{}, &KnativeKafkaList{})
 }
+
+// Used in s-o e2e tests. can't instantiate there because vendor dirs
+// of s-o and s-o/knative-operator are different
+func NewKnativeKafka(name, namespace string, spec KnativeKafkaSpec) *KnativeKafka {
+	return &KnativeKafka{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "KnativeKafka",
+			APIVersion: "operator.serverless.openshift.io/v1alpha1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: namespace,
+		},
+		Spec: spec,
+	}
+}
```

Comment:
Let's ignore this. Getting rid of this is an easy fix which can be done later.
Fix: https://github.com/openshift-knative/serverless-operator/pull/577/commits/370fc8530d15be95cdadf38c4f6ef4e5d88ae2d5

Reply:
PR against this PR's branch: https://github.com/matzew/serverless-operator/pull/3